packages/react/src/shared/tokens/space.ts

```diff
@@ -1,4 +1,4 @@
-const SPACE_VALUE = 4
+const SPACE_VALUE = 3
```

Comment:
Documenting here just to have the IRL convo written down: We should revert this back to 4 then use different spacing multiples to adjust individual components, so we can keep the spacing & font rhythms on the same scale.

Reply:
Fixed